Gruntfile.coffee

```diff
@@ -7,7 +7,7 @@ livereload = require 'tiny-lr'
 path = require 'path'
 fs = require 'fs'
 
-if process.env.LANGUAGE
+if process.env.DROPLET_LANGUAGE
```

Comment:
do you know what this environment variable is used for? it doesn't seem likely, but i'm wondering if there are any subsequent changes we'll need to make

Reply:
This was the only reference. `ignoredLanguages` is used at https://github.com/droplet-editor/droplet/blob/6251c62190970bebc1ba6b49bd5ccd33123cff2d/Gruntfile.coffee#L81 and with the naming collision was producing incomplete builds that resulted in our tests not being able to run.